sdk/nestjs/src/DevCycleModule/RequestInterceptor.ts

```diff
@@ -0,0 +1,32 @@
+import { Observable } from 'rxjs'
```

Comment:
is rxjs dependency just being added here in order to import a type?

Reply:
Yeah. The linter is unhappy if the response type of `intercept` isn't defined, and it's unhappy if rxjs is just added as a dev dependency. I supposed I could leave it as a dev dependency and just update linting config

Reply:
can you just type it as "unknown" ?

Reply:
or use `typeof TheThingYouAreReturning`

Reply:
Using unknown doesn't satisfy the NestInterceptor interface, and the thing we're returning is the result of `CallHandler.handle()` which is an rxjs Observable.

It's just the linting rule that wants exported response types to be explicitly defined. But considering typescript can infer the types just fine, and this interceptor is just used internally, we could ignore the rule for this case

Reply:
yeah just ignore it

Reply:
did ignoring it not work?

Reply:
I get a build error that the type can't be implied if I try to remove the return type and ignore this line. So instead I added rxjs as a dev dependency and update the lint config to ignore the dependency error for rxjs

Reply:
what about `ReturnType<CallHandler['handle']>`

Reply:
cool, that works!